wolfcrypt/benchmark/benchmark.c

```diff
@@ -198,6 +198,11 @@
 #ifdef WOLFCRYPT_HAVE_SAKKE
     #include <wolfssl/wolfcrypt/sakke.h>
 #endif
+#ifdef HAVE_LIBOQS
+    #include <wolfssl/wolfcrypt/falcon.h>
+    #include "falcon_level1_key.h"
```

Comment:
Would prefer having a macro in `wolfssl/certs_test.h` updated using `gencertbuf.pl`. Use a new macro with `HAVE_LIBOQS` in there...

Reply:
Will fix.